src/main/java/com/gluonhq/substrate/target/AndroidTargetConfiguration.java

```diff
@@ -533,4 +534,16 @@ private String getAndroidPackageName() {
         }
         return Collections.emptyList();
     }
+
+    /**
+     * Scans the classpath for Attach Services
+     * and extracts all aar libraries found
+     */
+    private void copyAarLibraries() throws IOException, InterruptedException {
+        Path libPath = getAndroidProjectPath().resolve("libs");
```

Comment:
what if this doesn't exist?

Reply:
That is taken care of by `FileOps.extractFilesFromJar`